sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamSuite.scala

```diff
@@ -1299,37 +1299,6 @@ class StreamSuite extends StreamTest {
     }
   }
 
-  test("SPARK-34482: correct active SparkSession for logicalPlan") {
-    withSQLConf(
-      SQLConf.ADAPTIVE_EXECUTION_ENABLED.key -> "true",
-      SQLConf.COALESCE_PARTITIONS_INITIAL_PARTITION_NUM.key -> "10") {
-      val df = spark.readStream.format(classOf[FakeDefaultSource].getName).load()
-      var query: StreamExecution = null
-      try {
-        query =
-          df.repartition($"a")
-            .writeStream
-            .format("memory")
-            .queryName("memory")
-            .start()
-            .asInstanceOf[StreamingQueryWrapper]
-            .streamingQuery
-        query.awaitInitialization(streamingTimeout.toMillis)
-        val plan = query.logicalPlan
-        val numPartition = plan
-          .find { _.isInstanceOf[RepartitionByExpression] }
-          .map(_.asInstanceOf[RepartitionByExpression].numPartitions)
-        // Before the fix of SPARK-34482, the numPartition is the value of
-        // `COALESCE_PARTITIONS_INITIAL_PARTITION_NUM`.
-        assert(numPartition.get === spark.sessionState.conf.getConf(SQLConf.SHUFFLE_PARTITIONS))
```

Comment:
Reason of deleting this test suite is the number of shuffle partition can change?

Reply:
Yes, the query is now eligible for AQE. So we are going back to the time where we didn't fix SPARK-34482.

> // Before the fix of SPARK-34482, the numPartition is the value of
> // `COALESCE_PARTITIONS_INITIAL_PARTITION_NUM`.